org.electrum.electrum.json

```diff
@@ -27,32 +27,28 @@
         "shared-modules/libusb/libusb.json",
         "libsecp256k1.json",
         {
-            "name": "libzbar",
-            "buildsystem": "autotools",
-            "config-opts": [
-                "--without-java",
-                "--without-gtk",
-                "--without-npapi",
-                "--without-xshm",
-                "--without-xv",
-                "--disable-video",
-                "--without-dbus",
-                "--without-imagemagick"
-            ],
-            "sources": [
-                {
-                    "type": "git",
-                    "url": "git://git.linuxtv.org/zbar.git",
-                    "tar": "0.23.1"
-                },
-                {
-                    "type": "script",
-                    "path": "autogen.sh",
-                    "commands": [
-                        "autoreconf -vfi -W none"
-                    ]
-                }
-            ]
+          "name": "zbar",
+          "config-opts": [
+            "--without-qt",
+            "--without-qt5",
+            "--without-gtk",
+            "--without-xv",
+            "--without-imagemagick",
+            "--without-dbus",
+            "--without-java",
+            "--without-npapi",
+            "--without-xshm",
+            "--with-python=python3",
+            "--disable-doc",
+            "--enable-codes=qrcode"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://www.linuxtv.org/downloads/zbar/zbar-0.23.tar.gz",
```

Comment:
0.23.1 is available.

Reply:
0.23.1 fails to build because of some error related to  gettext.h

Reply:
If I had to guess I would think it is related to gcc10 which is what the latest runtimes use.

Reply:
https://github.com/mchehab/zbar/issues/90

Reply:
It might also be a good idea to use the github source.

Reply:
updated, git source does not throw a gettex error 🤷‍♂